### ISSUE TITLE
Anchor header links are hidden to assistive technologies but are focusable

### ISSUE THREAD

**Issue by nickcolley** [alphagov/tech-docs-gem#126]
This may mean in some assistive technologies that anchor links are not announced.

## Impact HTML Markup
```html
<a href="#tech-docs-template" class="anchored-heading__icon" aria-hidden="true"></a>
```

## Failing WCAG Success Criteria
- [4.1.2: Name, Role, Value - Level A](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value)
